go/vt/vttablet/tabletserver/repltracker/poller.go

```diff
@@ -20,21 +20,35 @@ import (
 	"sync"
 	"time"
 
+	querypb "vitess.io/vitess/go/vt/proto/query"
+
+	"vitess.io/vitess/go/stats"
+
 	"vitess.io/vitess/go/vt/mysqlctl"
 	vtrpcpb "vitess.io/vitess/go/vt/proto/vtrpc"
 	"vitess.io/vitess/go/vt/vterrors"
 )
 
+var replicationLagGauges = stats.NewGaugesWithMultiLabels(
```

Comment:
None of the vttablet metrics have the keyspace/shard dimensions because by definition a tablet belongs to only one keyspace/shard. This can be a simple gauge (`NewGauge`).
Also the name should include the units - `replicationLagMs` or `replicationLagNs`. 
HeartbeatLag is being reported in nanoseconds so we should probably do the same here.

Reply:
Sure, I renamed the gauge to `replicationLagSec` since we always assume the lag in seconds (e.g., we have `SecondsBehindMaster` and also cast the duration to sec on line 60)

Reply:
That makes more sense than ns :)